test/button/button-subtle-perceptual.js

```diff
@@ -0,0 +1,67 @@
+const puppeteer = require('puppeteer');
+const visualDiff = require('visual-diff');
+
+visualDiff.run((ctx) => {
+
+	describe('d2l-button-subtle', function() {
+
+		this.timeout(10000);
+		let browser, page;
+
+		before(async() => {
+			browser = await puppeteer.launch();
+			page = await browser.newPage();
+		});
+
+		after(() => browser.close());
+
+		const runTests = () => {
+
+			it('normal', async function() {
+				const rect = await ctx.puppeteer.getRect(page, '#normal');
```

Comment:
The consumer can "stage" the fixture however it likes, and it can also choose to define it's own clip area if it wants, or even take a screenshot of the entire scrollable page as per the Puppeteer API.  This `getRect` helper just simplifies a common case of clipping a non-overflowing static or relatively positioned element.  They can optionally specify a margin to allow for box-shadow to be captured.